### ISSUE TITLE
Eslint errors when using eslint-loader with version beta.7

### ISSUE THREAD

**Issue by coreyleelarson** [pmmmwh/react-refresh-webpack-plugin#138]
It appears version 0.4.0-beta.7 is giving me a slew of eslint errors through eslint-loader. I have `enforce: 'pre'` set for eslint-loader to ensure it is executed on my untouched source files prior to babel-loader, but it seems like this plugin does not respect that.

I did not have this problem with version 0.4.0-beta.5 and everything was working great. Is there a new configuration that I am missing?

Let me know if you need any additional info.

Thanks for all your hard work!

**Example output:**
```
 error  in ./src/app/views/public/views/sponsor/index.js

Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

/home/corey/workspace/seeitmove/src/app/views/public/views/sponsor/index.js
   1:1   error  '$RefreshRuntime$' is not defined                                                                                                                                                               no-undef
   2:1   error  '$RefreshSetup$' is not defined                                                                                                                                                                 no-undef
   4:1   error  Import in body of module; reorder to top                                                                                                                                                        import/first
   5:1   error  Import in body of module; reorder to top                                                                                                                                                        import/first
  17:1   error  Delete `⏎`                                                                                                                                                                                      prettier/prettier
  18:24  error  '__react_refresh_utils__' is not defined                                                                                                                                                        no-undef
  19:1   error  '__react_refresh_utils__' is not defined                                                                                                                                                        no-undef
  19:56  error  Replace `currentExports,·module.id` with `⏎··currentExports,⏎··module.id,⏎`                                                                                                                     prettier/prettier
  25:7   error  '__react_refresh_utils__' is not defined                                                                                                                                                        no-undef
  36:8   error  Insert `,`                                                                                                                                                                                      prettier/prettier
  46:18  error  Identifier '__react_refresh_error_overlay__' is not in camel case                                                                                                                               camelcase
  47:11  error  Identifier '__react_refresh_error_overlay__' is not in camel case                                                                                                                               camelcase
  47:11  error  '__react_refresh_error_overlay__' is not defined                                                                                                                                                no-undef
  49:11  error  '__react_refresh_error_overlay__' is not defined                                                                                                                                                no-undef
  52:13  error  Replace `typeof·__react_refresh_test__·!==·'undefined'·&&·__react_refresh_test__` with `⏎··········typeof·__react_refresh_test__·!==·'undefined'·&&⏎··········__react_refresh_test__⏎········`  prettier/prettier
  52:20  error  Identifier '__react_refresh_test__' is not in camel case                                                                                                                                        camelcase
  52:62  error  Identifier '__react_refresh_test__' is not in camel case                                                                                                                                        camelcase
  52:62  error  '__react_refresh_test__' is not defined                                                                                                                                                         no-undef
  58:9   error  '__webpack_require__' is not defined                                                                                                                                                            no-undef
  59:8   error  Insert `,`                                                                                                                                                                                      prettier/prettier
  64:9   error  '__react_refresh_utils__' is not defined                                                                                                                                                        no-undef
  65:9   error  '__react_refresh_utils__' is not defined                                                                                                                                                        no-undef
  65:70  error  Replace `prevExports,·currentExports` with `⏎··········prevExports,⏎··········currentExports,⏎········`                                                                                         prettier/prettier
  69:9   error  '__react_refresh_utils__' is not defined                                                                                                                                                        no-undef
  76:22  error  Identifier '__react_refresh_error_overlay__' is not in camel case                                                                                                                               camelcase
  77:15  error  Identifier '__react_refresh_error_overlay__' is not in camel case                                                                                                                               camelcase
  77:15  error  '__react_refresh_error_overlay__' is not defined                                                                                                                                                no-undef
  79:15  error  '__react_refresh_error_overlay__' is not defined                                                                                                                                                no-undef
  81:12  error  Insert `,`                                                                                                                                                                                      prettier/prettier
  86:9   error  Replace `isHotUpdate·&&·__react_refresh_utils__.isReactRefreshBoundary(prevExports)` with `⏎······isHotUpdate·&&⏎······__react_refresh_utils__.isReactRefreshBoundary(prevExports)⏎····`        prettier/prettier
  86:24  error  '__react_refresh_utils__' is not defined                                                                                                                                                        no-undef
  90:2   error  Insert `⏎`                                                                                                                                                                                      prettier/prettier

✖ 32 problems (32 errors, 0 warnings)
  11 errors and 0 warnings potentially fixable with the `--fix` option.


 @ ./src/app/views/public/index.js 12:0-43 26:46-58
 @ ./src/app/index.js
 @ ./internals/entries/client.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js?sockPort=3001 ./internals/entries/client.js
```

**Relevant Webpack Config:**
```javascript
import ReactRefreshPlugin from '@pmmmwh/react-refresh-webpack-plugin';
import babelConfig from '../../.babelrc';

export default {
  module: {
    rules: [
      {
        exclude: /node_modules/,
        test: /\.js$/,
        loader: 'babel-loader',
        options: {
          babelrc: false,
          presets: babelConfig.presets,
          plugins: [
            ...babelConfig.plugins,
            ...(target === 'web'
              ? [require.resolve('react-refresh/babel')]
              : []),
          ],
        },
      },
      {
        enforce: 'pre',
        test: /\.js$/,
        loader: 'eslint-loader',
        options: {
          cache: true,
        },
      },
    ],
  },
  plugins: [
    new ReactRefreshPlugin({
      overlay: {
        sockPort: 3001,
      },
    }),
  ],
}
```

**Comment by coreyleelarson** [comment]
I think I've narrowed it down to #129 but I haven't had the time to investigate further.

**Comment by pmmmwh** [comment]
> I think I've narrowed it down to #129 but I haven't had the time to investigate further.

Yes, I am on it.

**Comment by pmmmwh** [comment]
Fixed in `0.4.0-beta.8`.

**Comment by coreyleelarson** [comment]
Works great, thanks!
